cognite/client/_api/data_modeling/instances.py

```diff
@@ -925,24 +933,23 @@ def apply(
                 >>> my_date = datetime(2020, 3, 14, 15, 9, 26, 535000, tzinfo=timezone.utc)
                 >>> data_model_timestamp = datetime_to_ms_iso_timestamp(my_date)  # "2020-03-14T15:09:26.535+00:00"
 
-            Create a typed node:
+            Create a typed node. Any property that you want to look up by a different attribute name, e.g. you want
+            `my_node.birth_year` to return the data for property `birthYear`, must use the PropertyOptions as shown below.
+            We strongly suggest you use snake_cased attribute names, as is done here:
 
                 >>> from cognite.client import CogniteClient
-                >>> from datetime import date
                 >>> from cognite.client.data_classes.data_modeling import TypedNodeApply, PropertyOptions
                 >>> class Person(TypedNodeApply):
-                ...     birth_date = PropertyOptions(identifier="birthDate")
+                ...     birth_year = PropertyOptions(identifier="birthYear")
```

Comment:
Perhaps `Person -> PersonApply`

Reply:
Yes, I think it is good to be consistent on that.